paddle/framework/op_registry.h

```diff
@@ -170,22 +174,33 @@ class OpKernelRegistrar : public Registrar {
 /**
  * Macro to register OperatorKernel.
  */
-#define REGISTER_OP_KERNEL(op_type, DEVICE_TYPE, place_class, ...)        \
-  STATIC_ASSERT_GLOBAL_NAMESPACE(                                         \
-      __reg_op_kernel_##op_type##_##DEVICE_TYPE##__,                      \
-      "REGISTER_OP_KERNEL must be called in global namespace");           \
-  static ::paddle::framework::OpKernelRegistrar<place_class, __VA_ARGS__> \
-      __op_kernel_registrar_##op_type##_##DEVICE_TYPE##__(#op_type);      \
-  int TouchOpKernelRegistrar_##op_type##_##DEVICE_TYPE() {                \
-    __op_kernel_registrar_##op_type##_##DEVICE_TYPE##__.Touch();          \
-    return 0;                                                             \
+#define REGISTER_OP_KERNEL(op_type, DEVICE_TYPE, place_class, priority, ...)  \
+  STATIC_ASSERT_GLOBAL_NAMESPACE(                                             \
+      __reg_op_kernel_##op_type##_##DEVICE_TYPE##_##priority##__,             \
+      "REGISTER_OP_KERNEL must be called in global namespace");               \
+  static ::paddle::framework::OpKernelRegistrar<place_class, priority,        \
+                                                __VA_ARGS__>                  \
+      __op_kernel_registrar_##op_type##_##DEVICE_TYPE##_##priority##__(       \
+          #op_type);                                                          \
+  int TouchOpKernelRegistrar_##op_type##_##DEVICE_TYPE##_##priority##__() {   \
+    __op_kernel_registrar_##op_type##_##DEVICE_TYPE##_##priority##__.Touch(); \
+    return 0;                                                                 \
   }
 
-#define REGISTER_OP_GPU_KERNEL(op_type, ...) \
-  REGISTER_OP_KERNEL(op_type, GPU, ::paddle::platform::GPUPlace, __VA_ARGS__)
+#define MIN_PRIORITY 0
```

Comment:
1. Do we need to explain the priorities mean? and 
2. When I use `REGISTER_OP_KERNEL`, what priority should I use?